frontend/common/stores/organisation-store.js

```diff
@@ -329,7 +329,11 @@ store.dispatcherIndex = Dispatcher.register(store, (payload) => {
 
   switch (action.actionType) {
     case Actions.GET_ORGANISATION:
-      controller.getOrganisation(action.id || store.id, action.force)
+      controller.getOrganisation(
+        action.id,
+        action.force || store.id,
```

Comment:
I think the function here only has 2 arguments, so the last is the typo.

```
    action.force || store.id,
```

I've read this through along with the function itself and I don't think it makes sense. The point of the force parameter is to hit the API regardless of whether we already have data for that org. By adding ||store.id here, we essentially totally remove that functionality?

Reply:
This change was reverted.